geometry_msgs/Pose goal_pose
geometry_msgs/Pose start_pose
bool is_goal_relative
---
bool answer
trajectory_msgs/JointTrajectory computed_trajectory